config/routes.rb

```diff
@@ -1,11 +1,11 @@
 Rails.application.routes.draw do
-  get 'static_pages/home'
-  get 'static_pages/help'
-  get 'static_pages/about'
-  get 'static_pages/aboutrtttt'
+  root "static_pages#home"
+  get  "/help", to: "static_pages#help"
```

Comment:
```suggestion
  get "/help", to: "static_pages#help"
```
sau get cách 1 space thôi em